pep-0692.rst

```diff
@@ -100,11 +99,19 @@ introduced.
 Specification
 =============
 
-To support the aforementioned use case we propose to use the double asterisk
-syntax inside of the type annotation. The required grammar change is discussed
-in more detail in section `Grammar Changes`_. Continuing the previous example::
+To support the aforementioned use case we propose to reuse ``Unpack`` which
+was initially introduced in :pep:`646`. There are several reasons for reusing
+:pep:`646`'s ``Unpack``. Firstly, the name is quite suitable and intuitive for
+the ``**kwargs`` typing use case as the keywords arguments are "unpacked" from
+the ``TypedDict``. Moreover, the current behavior of typing ``*args`` would be
+extended to ``**kwargs`` as well, which are supposed to behave similarly.
+Secondly, there would be no need to introduce any new special forms. Lastly,
+the use of ``Unpack`` for the purposes described in this PEP does not interfere
+with the use cases described in :pep:`646`.
```

Comment:
Yeah, I strongly second all of Adam's editorial recommendations. This long paragraph is unambiguously rationale, as it is listing the "reasons" for a particular choice as opposed to the choice itself, and thus is best suited to the Rationale section. Moreover, readers, editors and reviewers are best served by keeping the normative Specification section as succinct and to the point as practical, especially if this is eventually migrated to a dedicated Typing Specifications page/site, as if the PEP is accepted, those implementing the specification are primarily interested in just the specification itself, rather than wading through the rationale for every decision.

I'd also suggest a bulleted (unordered) list, since it makes a relatively dense prose paragraph easier to navigate, and avoids the need for imposing a quasi-arbitrary numerical order, which make have to be rewritten later if items are added, removed or re-ordered.

Reply:
Right, agreed, it makes more sense. I've moved the paragraph to the Rationale section and restructured it to a bullet point list.